PilotDataPlatform/helm-charts
# PILOT-6210: Enable open telemetry for search service

## Summary

- Enable open telemetry for search service

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-6210

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

- Verify search service open telemetry can be used with Tempo here: [https://monitoring-lab.indocpilot.io/explore?schemaVersion=1&panes={"sbs":{"datasource":"P214B5B846CF3925F","queries":[{"refId":"A","datasource":{"type":"tempo","uid":"P214B5B846CF3925F"},"queryType":"traceqlSearch","limit":20,"tableType":"traces","filters":[{"id":"fbc1387d","operator":"%3D","scope":"span"}]}],"range":{"from":"now-1h","to":"now"}}}&orgId=1](https://monitoring-lab.indocpilot.io/explore?schemaVersion=1&panes=%7B%22sbs%22:%7B%22datasource%22:%22P214B5B846CF3925F%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22datasource%22:%7B%22type%22:%22tempo%22,%22uid%22:%22P214B5B846CF3925F%22%7D,%22queryType%22:%22traceqlSearch%22,%22limit%22:20,%22tableType%22:%22traces%22,%22filters%22:%5B%7B%22id%22:%22fbc1387d%22,%22operator%22:%22%3D%22,%22scope%22:%22span%22%7D%5D%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D%7D&orgId=1)
